doc/src/getting-started-guide.md

```diff
@@ -137,203 +157,266 @@ When the AtomVM virtual machine starts, it will search for the first module that
+
+Note, however, that in general the USB device will auto-unmount after each file has been copied, so you will need to repeat the procedure for each of the above two files.
+
+On most Linux systems, the Raspberry Pico will be mounted at `/run/media/${USER}/RPI-RP2`.
 
-### Building AtomVM for Raspberry Pico
+On macOS system, the Raspberry Pico will be mounted at `/Volumes/RPI-RP2`.
 
-If you want to use a custom built VM for testing consult the [Build Instructions for Raspberry Pi Pico](./build-instructions#building-for-raspberry-pi-pico)
+For example:
 
-### Installing AtomVM and programs on Raspberry Pico
+    # power on Raspberry Pico with BOOTSEL button pressed
+    shell ls -l /Volumes/RPI-RP2
+    total 16
+    -rwxrwxrwx  1 joe  staff  241 Sep  5  2008 INDEX.HTM*
+    -rwxrwxrwx  1 joe  staff   62 Sep  5  2008 INFO_UF2.TXT*
 
-The approach consists in installing various uf2 files which include the
-address they should be loaded to.
+    shell$ cp ~/Downloads/AtomVM-pico-v0.6.0.uf2 /Volumes/RPI-RP2/.
+    ## at this point, the device will auto-unmount
 
-You typically need three uf2 files:
-- `AtomVM.uf2` for the VM
-- `atomvmlib.uf2` for the standard libraries
-- your application's uf2.
+And again for the AtomVM core library (note that previously flashed `.uf2` files have disappeared):
 
-We provide an escript-based (what else?) tool to build uf2 files called
-`uf2tool` that you can use to bundle your `avm` into uf2.
+    # power on Raspberry Pico with BOOTSEL button pressed
+    shell ls -l /Volumes/RPI-RP2
+    total 16
+    -rwxrwxrwx  1 joe  staff  241 Sep  5  2008 INDEX.HTM*
+    -rwxrwxrwx  1 joe  staff   62 Sep  5  2008 INFO_UF2.TXT*
 
-If you need to upgrade AtomVM or the standard libraries, simply copy them again.
+    shell$ cp ~/Downloads/atomvmlib-v0.6.0.avm /Volumes/RPI-RP2/.
+    ## at this point, the device will auto-unmount
 
-### Issues with macOS
+### Potential Issues with macOS
 
 There are known issues copying files to the Pico using macOS, and a lot of literature online. Usually it's best to use the Terminal rather than the Finder because the errors are more explicit.
 Copying may also fail with UF2 files downloaded from the Internet, typically AtomVM release binaries.
 
-```shell
-$ cp ~/Downloads/AtomVM-pico_w-v0.6.0-alpha1-test2.uf2 /Volumes/RPI-RP2/
-cp: /Volumes/RPI-RP2/AtomVM-pico_w-v0.6.0-alpha1-test2.uf2: fcopyfile failed: Operation not permitted
-cp: /Users/paul/Downloads/AtomVM-pico_w-v0.6.0-alpha1-test2.uf2: could not copy extended attributes to /Volumes/RPI-RP2/AtomVM-pico_w-v0.6.0-alpha1-test2.uf2: Operation not permitted
-```
+    shell$ cp ~/Downloads/AtomVM-pico_w-v0.6.0.uf2 /Volumes/RPI-RP2/.
+    cp: /Volumes/RPI-RP2/AtomVM-pico-v0.6.0-test2.uf2: fcopyfile failed: Operation not permitted
+    cp: /Users/paul/Downloads/AtomVM-pico-v0.6.0.uf2: could not copy extended attributes to /Volumes/RPI-RP2/AtomVM-pico-v0.6.0.uf2: Operation not permitted
```

Comment:
Lets do s/paul/joe/g here to be coherent :-)

Reply:
fixed :)